docs/data/data-grid/selection/CellSelectionGrid.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import { DataGridPremium } from '@mui/x-data-grid-premium';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+export default function CellSelectionGrid() {
+  const [rowSelection, setRowSelection] = React.useState(false);
+
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 10,
+    maxColumns: 6,
+  });
+
+  return (
+    <div style={{ width: '100%' }}>
+      <Button sx={{ mb: 2 }} onClick={() => setRowSelection(!rowSelection)}>
```

Comment:
This feels enough spacing:

```suggestion
      <Button sx={{ mb: 1 }} onClick={() => setRowSelection(!rowSelection)}>
```